### DISCUSSION TITLE
Problem preservating the Amiga version of "Seven Gates of Jambala" by Thalion/Grandslam

### DISCUSSION THREAD

**Discussion by realmml** [keirf/disk-utilities#140] [Q&A] [answered]
As usual disk-analyse let me to convert with success the most part of games from the last batch that my friends sent me, between the games we got:
Action Service, Double Dragon 1, Escape From The Planet of the Robot Monsters Spanish Edition, Final Assault, G.nius, Gear Works, Odyssey, Pirates, Z-Out (if you think that the raw dump of any of those games is interesting, I can upload them).

But aside of P47 or Ivanhoe, I found a really difficult one, the "Seven Gates of Jambala". I didn't find in the formats any other game by Thalion and because they were coming for Atari ST, I tried a few variations in the format, but no luck at all:
```
disk-analyse.exe -f probe_amiga convert\jambala\track jambala.ipf
T0.0: AmigaDOS
T0.1-79.1: Unformatted*
T80.0-83.1: Unformatted
** WARNING: 159 tracks are damaged or unidentified!
```
As you can see, only the boot sector is in AmigaDOS, the rest is using a copy protection by Thalion. You can donwload the dump here:
[Jambala Dump](https://www.dropbox.com/s/wdai2f1amcv3l46/jambala.zip)

**Comment by kkrellwitz** [comment]
I did spend some time on this and have created a decoder for track 15.0 and 158. Should have this fully tested soon.

**Reply by realmml** [reply]
Thanks Keith! :)

**Comment by kkrellwitz** [comment, marked as answer]
The repo has been updated and support for Seven Gates Of Jambal has been added.  Format is "Seven Gates Of Jambala"

**Reply by realmml** [reply]
Thanks!!! :D

**Comment by kkrellwitz** [comment]
Any chance of getting the raw dump of G.nius?

**Reply by realmml** [reply]
Of course, you can download here the raw dump:
https://www.dropbox.com/s/nxtchmkob1nlrn1/gnius_raw.zip?dl=0

For generating the ipf, I used this:
`$ ./disk-analyse.exe -v -f "G.Nius" ../convert/gnius/track gnius.ipf

Found format "G.Nius"
PLL Parameters: period_adj=5% phase_adj=60%
T0.0-49.1: AmigaDOS
T50.0-50.1: Lankhor Longtrack
T51.0-64.1: AmigaDOS
T65.0-83.1: Unformatted`

**Reply by realmml** [reply]
I don't know if it is interesting or not, but I should comment that the last few games that we have converted are King Quest I, II and III. The two first games have the herndon longtrack, but our original of KQ3 has not protection at all.
